compiler/src/dotty/tools/dotc/core/TypeErasure.scala

```diff
@@ -699,11 +746,12 @@ class TypeErasure(sourceLanguage: SourceLanguage, semiEraseVCs: Boolean, isConst
       }
     case _: ErrorType | JavaArrayType(_) =>
       tp
-    case tp: WildcardType if wildcardOK =>
+    case tp: WildcardType =>
+      assert(inSigName, i"Cannot erase wildcard type $tp")
+      null
+    case tp if (tp `eq` NoType) || (tp `eq` NoPrefix) => // Why is this check different?
```

Comment:
What do you mean by the question?

Reply:
Oops, that's a leftover stray comment I'll remove.